src/content/vagrant/install-vmware-landing.json

```diff
@@ -0,0 +1,38 @@
+{
+	"featuredTutorialsSlugs": [
+		"vagrant/getting-started-index",
+		"vagrant/getting-started-boxes",
+		"vagrant/getting-started-networking"
+	],
+	"packageManagerOverrides": [],
```

Comment:
Should I adjust this so the package manager codeblock doesn't render? Is it possible to do that via the content?

Reply:
Looks like we can use `doesNotHavePackageManagers`, per https://github.com/hashicorp/dev-portal/tree/main/src/content#5-product-install-landing-pages

Reply:
ohhh 💯